convertCSV.js

```diff
@@ -0,0 +1,50 @@
+class CsvPhraseImporter {
+  constructor(event) {
+    this.readFile(event)
+  }
+  
+  readFile(event) {
+    let file = event.target.files[0]
+    
+    const reader = new FileReader();
+    reader.addEventListener('load', (event) => {
+      //console.log(csv)
+      this.phrase = this.csvToPhrase(event.target.result)
+    });
+    reader.readAsText(file);
+  }
+  
+  toPhrase = function(value, state) {
+    console.log("hello?")
+    switch (state.colNum) {
+      case 1:
+        var word = value
+        return false
+      case 2:
+        var hint = value
+        return false
+      case 3:
+        var category = value
+        return new Phrase(word, hint, category)
+      default:
+        console.log("entered default of toPhrase function")
+        return false
+    }
+  }
+  logPhraseEntries = function(data) {
```

Comment:
```suggestion
  logPhraseEntries(data) {
```

Reply:
Switched to a different parser library. Callback function was deleted as it was no longer required.